docs/getting-started/branching.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Creating a Feature Branch
+---
+
+# Creating a Feature / Bug Branch
+
+When you are working on a bug, feature or improvement, you will need to create a branch. For features or improvement you will create a branch as follows:
```

Comment:
I think "you will" is a bit imperative :D Being a suggestion I'd replace it with "you could"